playbook-examples/manage_cnp.yml

```diff
@@ -0,0 +1,106 @@
+# Original work Copyright 2019-2020, EnterpriseDB Corporation
+#
+# Installs Roles and Collections requirements
+# ansible-galaxy install --force -r requirements.yml
+#
+# pip install openshift pyyaml kubernetes
+# pip install pyhelm
+---
+- name: Manage a CloudNativePG or CloudNative Postgres Cluster Playbook
+  hosts: localhost
+  gather_facts: false
+
+  vars:
+    # Options"
+    # add-db, add-table, add-index, add-view
+    # drop-db, drop-table, drop-index, drop-view
+    - cnp_task: "drop-db"
+
+  roles:
+    - role: manage_cnp
+      include_tasks: add_db.yml
+      when: cnp_task == "add-db"
+
+    - role: manage_cnp
+      include_tasks: add_schema.yml
+      when: cnp_task == "add-schema"
+
+    - role: manage_cnp
+      include_tasks: add_role.yml
+      when: cnp_task == "add-role"
+
+    - role: manage_cnp
+      include_tasks: add_table.yml
+      when: cnp_task == "add-table"
+
+    - role: manage_cnp
+      include_tasks: add_index.yml
+      when: cnp_task == "add-index"
+
+    - role: manage_cnp
+      include_tasks: add_view.yml
+      when: cnp_task == "add-view"
+
+    - role: manage_cnp
+      include_tasks: add_trigger.yml
+      when: cnp_task == "add-trigger"
+
+    - role: manage_cnp
+      include_tasks: add_sequence.yml
+      when: cnp_task == "add-sequence"
+
+    - role: manage_cnp
+      include_tasks: add_type.yml
+      when: cnp_task == "add-type"
+
+    - role: manage_cnp
+      include_tasks: add_procedure.yml
+      when: cnp_task == "add-procedure"
+
+    - role: manage_cnp
+      include_tasks: add_function.yml
+      when: cnp_task == "add-function"
+
+    - role: manage_cnp
+      include_tasks: drop_function.yml
+      when: cnp_task == "drop-function"
+
+    - role: manage_cnp
+      include_tasks: drop_procedure.yml
+      when: cnp_task == "drop-procedure"
+
+    - role: manage_cnp
+      include_tasks: drop_type.yml
+      when: cnp_task == "drop-type"
+
+    - role: manage_cnp
+      include_tasks: drop_sequence.yml
+      when: cnp_task == "drop-sequence"
+
+    - role: manage_cnp
+      include_tasks: drop_trigger.yml
+      when: cnp_task == "drop-trigger"
+
+    - role: manage_cnp
+      include_tasks: drop_view.yml
+      when: cnp_task == "drop-view"
+
+    - role: manage_cnp
+      include_tasks: drop_index.yml
+      when: cnp_task == "drop-index"
+
+    - role: manage_cnp
+      include_tasks: drop_table.yml
+      when: cnp_task == "drop-table"
+
+    - role: manage_cnp
+      include_tasks: drop_role.yml
+      when: cnp_task == "drop-role"
+
+    - role: manage_cnp
+      include_tasks: drop_schema.yml
+      when: cnp_task == "drop-schema"
+
+    - role: manage_cnp
+      include_tasks: drop_db.yml
+      when: cnp_task == "drop-db"
```

Comment:
Using the `include_tasks` module would need to be defined under a `tasks` header instead of `role`. With this being similar to the `manage_cnp/tasks/main.yml`, I think defining 
```
- role: manage_cnp
  when: cnp_task is defined # or some other conditional
```
would be more appropriate for a playbook.

Reply:
Fixed